frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -144,28 +135,28 @@ function getParameterPanelBorderColor(theme?: DisplayTheme) {
 
 export const ParametersWidgetContainer = styled(FullWidthContainer)<{
   embedFrameTheme?: DisplayTheme;
-  canSticky: boolean;
+  allowSticky: boolean;
   isSticky: boolean;
 }>`
   padding-top: ${space(1)};
   padding-bottom: ${space(1)};
 
-  ${props =>
-    props.canSticky &&
+  ${({ allowSticky }) =>
+    allowSticky &&
     css`
       position: sticky;
       top: 0;
       left: 0;
       width: 100%;
       z-index: 3;
-      transition: background-color 0.4s;
-      background-color: ${getParameterPanelBackgroundColor(
-        props.embedFrameTheme,
-        props.isSticky,
-      )};
-      border-bottom: ${props.isSticky &&
-      `1px solid
-        ${getParameterPanelBorderColor(props.embedFrameTheme)}`};
+      border-bottom: 1px solid transparent;
+    `}
+
+  ${({ embedFrameTheme, isSticky }) =>
+    isSticky &&
+    css`
+      background-color: ${getParameterPanelBackgroundColor(embedFrameTheme)};
```

Comment:
We set the bg color in the sticky state only now, so no need to pass a second parameter